src/Service.Tests/DatabaseSchema-MsSql.sql

```diff
@@ -47,6 +47,8 @@ DROP TABLE IF EXISTS graphql_incompatible;
 DROP TABLE IF EXISTS GQLmappings;
 DROP TABLE IF EXISTS bookmarks;
 DROP TABLE IF EXISTS mappedbookmarks;
+DROP TABLE IF EXISTS fte_data;
+DROP TABLE IF EXISTS intern_data;
```

Comment:
do you need to drop the triggers before SQL will let you drop the tables?

Reply:
No, we don't.